.github/workflows/build-commit.yaml

```diff
@@ -1,6 +1,24 @@
 name: build-commit
 
 on:
+  workflow_call:
+    inputs:
+      arch:
+        description: The machine architecture to build for
+        required: true
+        type: string
+      python_version:
+        description: The version of Python to use
+        required: true
+        type: string
+    secrets:
+      ACTIONS_AWS_ROLE_ARN:
+        description: The AWS Role ARN for secrets access
+        required: true
```

Comment:
Why is this required when it wasn't before?

Reply:
It's required when the workflow is invoked as a dependent workflow. Here is the PR that failed when `build-commit` didn't contain that:

https://github.com/Eventual-Inc/Daft/actions/runs/12403362351

Reply:
I see, ok yeah we should check also that this doesn't break just running the build-commit workflow by itself from the UI or anything

Reply:
Ran one right now. Looks all good. You can observe the workflow file contains the `secrets` input as well.

https://github.com/Eventual-Inc/Daft/actions/runs/12404404099